Update v3.26 release notes (#180)
gradebook/marketing: src/release-notes.md

_Note that we only update our release notes when there is a notable feature or improvement. Between the documented releases here, we can have several smaller releases with bug fixes or improvements_ 😊

## 3.26.2

_Released August 16, 2022_

_Even though there aren't notable features or improvements, we've decided to include the changes from 3.26.0 to 3.26.2 due to reliability issues._

## Bug Fixes

1. Importing a course should now be fully functional

 - Loading the import page will not crash

 - Trying to import a course after logging in will not cause a crash

> 2. Logging in to Gradebook as a student in a different school will work

Reviewer: ```suggestion
2. Logging in to a school's site on Gradebook will properly take you to the correct school
```

Author: a `different` school on Gradebook will take you to `your` correct school?

Author: Whatever you decide, please do a quick UI review before merging!

Reviewer: I like your wording

 - Previously, the app would try to log you in, fail, and ask you to log in again

3. Account creation after trying to import a course will work now

>  - Previously, you could get a cryptic error

Reviewer: /n

> 3. Account creation after trying to import a course will work now
>  - Previously, you could get a cryptic error

Reviewer: ```suggestion
3. Account creation after trying to import a course will work now

 - Previously, you could get a cryptic error
```

- Upcoming semesters can now be created much earlier

- The semester used for GPA will always be the oldest of current semesters.

>   - During transition periods, such as between the Fall and Winter semester, the Fall semester will be used for GPA calculations until it is archived.

Reviewer: ```suggestion
  - During transition periods, such as between the Fall and Winter semester, the older semester will be used for GPA calculations until it is archived.
```
